images/manageiq-base/Dockerfile

```diff
@@ -57,108 +37,29 @@ LABEL name="manageiq-base" \
       io.openshift.tags="ManageIQ,miq,manageiq"
 
 RUN curl -L https://copr.fedorainfracloud.org/coprs/manageiq/ManageIQ-Master/repo/epel-8/manageiq-ManageIQ-Master-epel-8.repo > /etc/yum.repos.d/manageiq.repo && \
+    curl -L https://copr.fedorainfracloud.org/coprs/simaishi/test/repo/epel-8/simaishi-test-epel-8.repo > /etc/yum.repos.d/simaishi.repo && \
     curl -L https://releases.ansible.com/ansible-runner/ansible-runner.el8.repo > /etc/yum.repos.d/ansible-runner.repo
 
 RUN dnf -y --disableplugin=subscription-manager install http://mirror.centos.org/centos/8/BaseOS/${ARCH}/os/Packages/centos-repos-8.1-1.1911.0.8.el8.${ARCH}.rpm http://mirror.centos.org/centos/8/BaseOS/${ARCH}/os/Packages/centos-gpg-keys-8.1-1.1911.0.8.el8.noarch.rpm https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm && \
     dnf -y --disableplugin=subscription-manager module enable nodejs:12 && \
     dnf -y --disableplugin=subscription-manager module enable ruby:2.6 && \
     dnf config-manager --setopt=ubi-8-appstream.exclude=*net-snmp* --setopt=ubi-8-baseos.exclude=*net-snmp* --setopt=ubi-8-codeready-builder.exclude=*net-snmp* --save && \
     dnf -y --disableplugin=subscription-manager --setopt=tsflags=nodocs install \
-      ansible                     \
-      ansible-runner              \
-      cmake                       \
-      cockpit-ws                  \
-      cockpit-ssh                 \
-      cyrus-sasl                  \
-      cyrus-sasl-plain            \
       file                        \
-      freeipmi                    \
-      gcc-c++                     \
-      git                         \
-      glibc-langpack-en           \
-      hostname                    \
-      http-parser                 \
-      ipmitool                    \
-      iproute                     \
-      libcurl-devel               \
-      libpq                       \
-      libssh2-devel               \
-      libtool                     \
-      libxml2-devel               \
-      libxslt-devel               \
-      logrotate                   \
       lvm2                        \
-      make                        \
-      net-snmp                    \
-      net-snmp-libs               \
-      net-snmp-utils              \
-      nmap-ncat                   \
-      nodejs                      \
-      openldap-clients            \
-      openscap-scanner            \
-      openssl-devel               \
-      OpenIPMI                    \
+      manageiq-pods               \
       patch                       \
-      postfix                     \
-      postgresql-devel            \
-      procps-ng                   \
       python3-devel               \
-      qpid-proton-c-devel         \
-      redhat-rpm-config           \
-      ruby-devel                  \
-      socat                       \
-      sqlite-devel                \
-      v2v-conversion-host-ansible \
       wget                        \
```

Comment:
@simaishi I wonder if we should create a manageiq-pods-tools rpm analogous to the manageiq-appliance-tools rpm that houses stuff like wget and patch here.  What do you think? (not for this PR, we can iterate on that later)

Reply:
I'm not sure where file, lvm2, patch and wget are used, so I left them here for now. I'm hoping all those will go to one of the manageiq-* RPM "requires". If they are just tools and not required to run our app, we can certainly create manageiq-pods-tools.

Reply:
@carbonin do you know?  I assumed patch and wget were just convenience tools.

Reply:
`wget` is used here https://github.com/ManageIQ/manageiq-pods/pull/485/files#diff-4f1c0cb40242780de19fd44d5c011b0eL167 but I suppose we could use `curl`.

No idea where `lvm2` or `patch` are used (if at all)

Reply:
Ah yeah, if we can use curl there and avoid adding a package I'm :+1: for that

Reply:
Can we just try dropping lvm2 and patch and see what happens?

Reply:
Also noticed `file`

Reply:
> Can we just try dropping lvm2 and patch and see what happens?

I'm okay with that.

Reply:
Removed file, lvm2, patch and wget (But file is installed as a dependency anyway).